modules/integration/tests-integration/tests-backend/src/test/resources/rest-api-test-data/ApplicationWithCustomAttributesTestCase.txt

```diff
@@ -0,0 +1,143 @@
+{
+	"testCase": [
+	    {
+			"init": {
+				"scope": "apim:subscribe"
+			},
+			"data": {
+				"method": "POST",
+				"url": "/api/am/store/v0.12//applications",
```

Comment:
Please change v0.12 to v0.13. check other places as well.

Reply:
Resolved